core/src/main/java/com/google/instrumentation/trace/Tracer.java

```diff
@@ -31,14 +31,21 @@
  * <p>Users may choose to use manual or automatic Context propagation. Because of that this class
  * offers APIs to facilitate both usages.
  *
+ * <p>The automatic context propagation is done using {@link io.grpc.Context} which is a gRPC
```

Comment:
too bad the package name of Context can't be changed from grpc, as might raise questions over time.